doc/release-notes.md

```diff
@@ -110,6 +110,7 @@ See detailed [set of changes](https://github.com/dashpay/dash/compare/v0.13.1.0.
 - [`9e233f391`](https://github.com/dashpay/dash/commit/9e233f391) Fix incorrect usage of begin() when genesis block is requested in "protx diff" (#2699)
 - [`e75f971b9`](https://github.com/dashpay/dash/commit/e75f971b9) Do not process blocks in CDeterministicMNManager before dip3 activation (#2698)
 - [`1cc47ebcd`](https://github.com/dashpay/dash/commit/1cc47ebcd) Backport #14701: build: Add CLIENT_VERSION_BUILD to CFBundleGetInfoString (#2687)
+- [`548a48918`](https://github.com/dashpay/dash/commit/548a48918) Move IS block filtering into ConnectBlock (#2766)
```

Comment:
This line should be at the top of "Backports" as the most recent one. And while at it, the line with `Bump version to 0.13.2` should really be in "Other", not in "Backports" 🙈